api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -9,8 +21,46 @@
 @Service
 public class MailServiceImpl implements MailService {
 
+    private final Provider<MandrillApi> mandrillApiProvider;
+    private final Provider<CloudStorageService> cloudStorageServiceProvider;
+
+    @Autowired
+    public MailServiceImpl(Provider<MandrillApi> mandrillApiProvider,
+                               Provider<CloudStorageService> cloudStorageServiceProvider) {
+        this.mandrillApiProvider = mandrillApiProvider;
+        this.cloudStorageServiceProvider = cloudStorageServiceProvider;
+    }
+
     public void send(Message msg) throws MessagingException {
         Transport.send(msg);
     }
 
+    public MandrillMessageStatuses sendEmail(String contactEmail, String password, User user) throws MessagingException {
+        MandrillApi mandrillApi = mandrillApiProvider.get();
+        String apiKey = cloudStorageServiceProvider.get().readMandrillApiKey();
+        MandrillApiKeyAndMessage keyAndMessage = new MandrillApiKeyAndMessage();
+        keyAndMessage.setKey(apiKey);
+        MandrillMessage msg = buildMandrillMessage(contactEmail, password, user);
+        keyAndMessage.setMessage(msg);
+        try {
+            MandrillMessageStatuses msgStatuses = mandrillApi.send(keyAndMessage);
+            return msgStatuses;
+        } catch (ApiException e){
+            throw new MessagingException("Sending email failed.");
+        }
+    }
+
+    private MandrillMessage buildMandrillMessage(String contactEmail, String password, User user) {
```

Comment:
Similar to the comment about the generic-ness of the interface's method name, I think this, too, is too generic. It should be very clear that this method is building a welcome to AoU email message.